tests/foreman/ui/test_ansible.py

```diff
@@ -0,0 +1,121 @@
+"""Test class for Ansible Roles and Variables pages
+
+:Requirement: Ansible
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Ansible
+
+:Assignee: dsynk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from airgun.session import Session
+
+from robottelo.datafactory import gen_string
+
+
+def test_positive_import_all_roles(target_sat):
+    """Import all Ansible roles available by default.
+
+    :id: 53fe3857-a08f-493d-93c7-3fed331ed391
+
+    :Steps:
+
+        1. Navigate to the Configure > Roles page.
+        2. Click the `Import from [hostname]` button.
+        3. Get totally number of importable roles from pagination?
+        4. Fill the `Select All` checkbox.
+        5. Click the `Submit` button.
+        6. Verify that number of imported roles == number of importable roles from step 3.
+        7. Verify that Ansible variables have been imported along with roles.
+
+    :expectedresults: All roles are imported successfully.
+    """
+    with Session(hostname=target_sat.hostname) as session:
+        available_roles_count = session.ansibleroles.import_all_roles()
+        imported_roles_count = session.ansibleroles.imported_roles_count
+        assert available_roles_count == imported_roles_count
+        ansible_variables_count = int(session.ansiblevariables.read_total_variables())
+        assert ansible_variables_count > 0
+        delete_role = 'theforeman.foreman_scap_client'
+        session.ansibleroles.delete(delete_role)
+        assert not session.ansibleroles.search(delete_role)
```

Comment:
Optional: you might want to optimize this a bit
```suggestion
        assert session.ansibleroles.import_all_roles() == session.ansibleroles.imported_roles_count
        assert int(session.ansiblevariables.read_total_variables()) > 0
        session.ansibleroles.delete('theforeman.foreman_scap_client')
        assert not session.ansibleroles.search('theforeman.foreman_scap_client')
```